core/api/src/domain/bitcoin/lightning/index.types.d.ts

```diff
@@ -110,7 +110,7 @@ type LnInvoice = {
   readonly paymentSecret: PaymentIdentifyingSecret | null
   readonly features: LnInvoiceFeature[]
   readonly expiresAt: Date
-  readonly isExpired: boolean
+  readonly isExpired: boolean // Should we remove this because it can become stale?
```

Comment:
I think this can stay for now since we use it in a live sort of way when we `decodeInvoice` and assign the value against time at function call